AK/String.cpp

```diff
@@ -230,6 +230,9 @@ void String::destroy_string()
 
 ErrorOr<String> String::from_utf8(StringView view)
 {
+    if (!Utf8View { view }.validate())
```

Comment:
(looks like this snippet could be extracted and reused in L145-146? not sure how much is it needed, though – it seems it could even find itʼs way to `String.h:76` somehow – maybe it should just go to `validate`!? 🤔)

Reply:
seems fine, not sure why you'd want to further DRY a single function call.